cmake/README.md

```diff
@@ -20,6 +20,7 @@ TBB_VALGRIND_MEMCHECK:BOOL - Enable scan for memory leaks using Valgrind (OFF by
 TBB_DISABLE_HWLOC_AUTOMATIC_SEARCH - Disable HWLOC automatic search by pkg-config tool (OFF by default)
 TBB_ENABLE_IPO - Enable Interprocedural Optimization (IPO) during the compilation (ON by default)
 TBB_BUILD_APPLE_FRAMEWORKS - Enable the Apple* frameworks instead of dylibs, only available on the Apple platform. (OFF by default)
+TBB_FILE_TRIM - Enable __FILE__ trim (ON by default)
```

Comment:
To be honest, "Enable FILE trim" still doesn't say much about what this option does. Can you add more description about usage of relative paths instead of full path.

Reply:
More description has been added.

Reply:
@aepanchi Could you please take a look?